src/Augurk.Api/Controllers/V2/AugurkController.cs

```diff
@@ -75,5 +87,139 @@ public async Task PersisConfigurationAsync(Configuration configuration)
         {
             await _configurationManager.PersistConfigurationAsync(configuration);
         }
+
+        /// <summary>
+        /// Imports existing data into Augurk.
+        /// </summary>
+        /// <returns></returns>
+        [Route("import")]
+        [HttpPost]
+        public async Task<HttpResponseMessage> Import()
+        {
+            // Make sure that we actually got the right data
+            if (!Request.Content.IsMimeMultipartContent())
+            {
+                throw new HttpResponseException(HttpStatusCode.UnsupportedMediaType);
+            }
+
+            try
+            {
+                // Store the uploaded file into a temporary location
+                var provider = new MultipartFormDataStreamProvider(Path.GetTempPath());
+                await Request.Content.ReadAsMultipartAsync(provider);
+
+                string filename = provider.FormData.GetValues("filename").First();
+                var file = provider.FileData.First();
+
+                // Determine the appropriate import method to use
+                SmugglerDatabaseApiBase importer;
+                RavenConnectionStringOptions connectionStringOptions;
+                if (Database.DocumentStore is EmbeddableDocumentStore embeddableDocumentStore)
+                {
+                    importer = new DatabaseDataDumper(embeddableDocumentStore.DocumentDatabase);
+                    connectionStringOptions = new EmbeddedRavenConnectionStringOptions();
+                }
+                else
+                {
+                    importer = new SmugglerDatabaseApi();
+                    connectionStringOptions = new RavenConnectionStringOptions()
+                    {
+                        Url = Database.DocumentStore.Url
+                    };
+                }
+
+                // Setup an import using RavenDb's Smuggler API
```

Comment:
The comment doesn't really reflect what we're doing here.
Isn't this step configuring the import?